salt/modules/statuspage.py

```diff
@@ -0,0 +1,553 @@
+# -*- coding: utf-8 -*-
+'''
+StatusPage
+==========
+
+Handle requests for the StatusPage_ API_.
+
+.. _StatusPage: https://www.statuspage.io/
+.. _API: http://doers.statuspage.io/api/v1/
+
+In the minion configuration file, the following block is required:
+
+.. code-block:: yaml
+
+  statuspage:
+    api_key: <API_KEY>
+    page_id: <PAGE_ID>
+
+.. versionadded:: Nitrogen
+'''
+
+from __future__ import absolute_import
+
+# import python std lib
+import logging
+
+# import third party
+import requests
```

Comment:
Isn't `requests` one of the requirements? https://github.com/saltstack/salt/blob/develop/requirements/base.txt#L5